js/repl/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
```

Comment:
In theory the build process for the JS could be whatever you wanted it to be, as long as the files end up in `_site`.